geograypher/meshes/meshes.py

```diff
@@ -2155,12 +2158,28 @@ def save_renders(
 
             # This may create nested folders in the output dir
             ensure_containing_folder(output_filename)
-            if rendered.dtype == np.uint8:
-                output_filename = str(output_filename.with_suffix(".png"))
 
-                # Save the image
-                skimage.io.imsave(output_filename, rendered, check_contrast=False)
-            else:
+            if save_as_npy is True:
                 output_filename = str(output_filename.with_suffix(".npy"))
                 # Save the image
                 np.save(output_filename, rendered)
+            else:
+                # Save image as TIF
+                output_filename = str(output_filename.with_suffix(".tif"))
+                rendered = np.squeeze(rendered)
+                # If cast_to_uint8 is True, rendered is already in uint8
+                if cast_to_uint8 is False:
+                    # Check if max value in the rendered image is within the range of uint16
```

Comment:
One thing I just realized, we could now use tif for float data that actually should be treated as float, like CHM renders. But that would probably need another flag and I'm not sure if it would be a space savings compared to npy. Maybe save that change for later if we need it?